### ISSUE TITLE
Uncaught TypeError: window.data.move is undefined (NUXT JS)

### ISSUE THREAD

**Issue by gitamgadtaula** [nil/v-drag#27]
I have been using v-drag with nuxt js SPA.

plugins/drag.js
```
import Vue from "vue";
import drag from "v-drag";
Vue.use(drag);
```
But it says
```

Uncaught TypeError: window.data.move is undefined
    default moveElementTransform.js:2
    default dragEnd.js:16
    default eventListener.js:4
    default eventListener.js:3
    default dragSetup.js:46
    inserted index.js:36
    VueJS 20
```

I have tried so many ways but still no help. At this point I'm just thinking to change the library itself but I find it so easy and lightweight. So here I am taking my chances to see if anyone can come up with a solution. Cheers !!

**Comment by kunalbhat** [comment]
@gitamgadtaula did you ever figure out a workaround for this issue?

**Comment by kunalbhat** [comment]
It just occurred to me that this error is likely related to SSR in my app, I was able to get everything work by wrapping the `Vue.use(drag);` line with a check to handle this client-side only.

```
if (process.browser) {
  Vue.use(drag);
}
```

**Comment by nil** [comment]
I believe this issue was fixed with pull request #23. Can you confirm you are not getting this error?

**Comment by gitamgadtaula** [comment]
Hi @nil , I moved on from this library quite a while ago, we couldn't afford to take it into our production server back then, but I hope now it works well and will be helpful to anyone in the future. Cheers !!
